File: parantier-front/src/entities/work/types/workFigma.ts
export interface WorkFigma {
  id: number
  workId: number
  title: string
  url: string
  description?: string
  orderNum: number
  createdAt: string
  updatedAt: string
}

export interface CreateWorkFigmaRequest {
  title: string
  url: string
  description?: string
  orderNum?: number
}

export interface UpdateWorkFigmaRequest {
  title: string
  url: string
  description?: string
  orderNum: number
}

File: parantier-api/src/main/java/com/mapo/palantier/work/domain/WorkFigma.java
package com.mapo.palantier.work.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WorkFigma {
    private Long id;
    private Long workId;
    private String title;
    private String url;
    private String description;
    private Integer orderNum;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
